include/dt-bindings/gpio/gpio.h

```diff
@@ -81,6 +81,7 @@
  *
  * Following defines are deprecated and shouldn't be used in DTS files.
  */
+#define GPIO_DIR_IN             (0 << 9) /* GPIO_INPUT */
```

Comment:
It should be mapped to `(1 << 8)`, i.e. has to have the same value as an existing `GPIO_INPUT` flag in `include/drivers/gpio.h`.

Reply:
done